packages/sdk-react/src/components/receivables/InvoiceDetails/ExistingInvoiceDetails/components/ReceivableRecurrence/InvoiceRecurrenceForm.tsx

```diff
@@ -228,6 +228,8 @@ export const InvoiceRecurrenceForm = ({
                       end_year,
                       start_month,
                       start_year,
+                      body_text: '', // TODO: add support for body_text
```

Comment:
This shouldn't be a part of this PR

Reply:
It should be, I'm updating the API version there is no way to not have it as part of this PR